server/cmd/mmctl/commands/channel.go

```diff
@@ -368,18 +368,24 @@ func unarchiveChannelsCmdF(c client.Client, cmd *cobra.Command, args []string) e
 		return errors.New("enter at least one channel")
 	}
 
+	var errs *multierror.Error
+
 	channels := getChannelsFromChannelArgs(c, args)
 	for i, channel := range channels {
 		if channel == nil {
-			printer.PrintError("Unable to find channel '" + args[i] + "'")
+			errs = multierror.Append(errs, errors.New("Unable to find channel '"+args[i]+"'"))
 			continue
 		}
 		if _, _, err := c.RestoreChannel(context.TODO(), channel.Id); err != nil {
-			printer.PrintError("Unable to unarchive channel '" + args[i] + "'. Error: " + err.Error())
+			errs = multierror.Append(errs, errors.New("Unable to unarchive channel '"+args[i]+"'. Error: "+err.Error()))
 		}
 	}
 
-	return nil
+	for _, err := range errs.WrappedErrors() {
```

Comment:
@hanzei 

Hmm. Actually in the issue, the idea was to use multierror & return the errors after the loop. And I think that stopping the loop right away might be more tedious. 
(Fix, Run, Fix, Run)

Since we're using multierror. Can we just remove this loop & return the error right away? that might be better

Reply:
@hanzei Updated code